ai-engineering/evaluate/run-evaluations.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Run evaluations"
+description: "Learn how to run evaluations using the Axiom CLI and interpret the results."
+keywords: ["ai engineering", "cli", "run evals", "commands", "testing"]
+---
+
+The Axiom AI SDK CLI provides commands for running evaluations locally or in CI/CD pipelines.
+
+## Run evaluations
+
+The simplest way to run evaluations is to execute all of them in your project:
+
+```bash
+axiom eval
+```
+
+You can also target specific evaluations by name, file path, or glob pattern:
+
+```bash
+# By evaluation name
+axiom eval spam-classification
+
+# By file path
+axiom eval src/evals/spam-classification.eval.ts
+
+# By glob pattern
+axiom eval "**/*spam*.eval.ts"
+```
+
+To see which evaluations are available without running them:
+
+```bash
+axiom eval --list
+```
+
+## Common options
+
+During development, use watch mode to automatically re-run evaluations when files change:
+
+```bash
+axiom eval --watch
+```
```

Comment:
i dont know if we should advertise this. watch mode is a lot less useful when it takes 5 minutes to run and costs $3